max/kernels/src/layout/tensor_core.mojo

```diff
@@ -173,7 +174,7 @@ struct TensorCore[
     # Layout reference => https://github.com/NVIDIA/cutlass/blob/main/include/cute/atom/mma_traits_sm80.hpp#L44.
 
     alias supported_fp32 = in_type is DType.float32 and (
-        shape == shape_16x8x8 if is_nvidia_gpu() else shape == shape_16x16x4
+        shape == shape_16x8x8 if is_nvidia_gpu() else shape == shape_16x16x16
```

Comment:
This might break CDNA, and I don't think it's necessary to get matmul tests working correctly on RDNA. I reverted this change and was still able to get everything to build on RNDA correctly.